client/src/app/api/rest.ts

```diff
@@ -463,7 +480,7 @@ export const createFile = ({
   file: IReadFile;
 }) =>
   axios
-    .post<Ruleset>(`${FILES}/${file.fileName}`, formData, fileHeaders)
```

Comment:
Why remove the type param here? Is there a suitable replacement or is inference enough here?

Reply:
Added a new type for HubFile here. Fixed.